### ISSUE TITLE
Battery power max in Basic settings with 48V 750W battery & 500W motor?

### ISSUE THREAD

**Issue by kiwisibk** [emmebrusa/TSDZ2-Smart-EBike-1#82]
As the subject says, with a 48V 750W battery, what value should I enter in "Battery power max" in "Basic settings" when using a 500W motor?

Thank you

**Comment by dzid26** [comment]
It's just electrical power. It's somewhat meaningless because it is not mechanical power which depends on motor speed and motor characteristics. 
From a hardware protection point of view, this value also doesn't matter directly. Maximum current is what matters (both for the motor and the battery)! 
Since battery voltage is somewhat stable, this value is basically another way to limit the current. I guess.  For people who perhaps know rated battery pack power, but don't know the max current... or Ohm's law. I = P / U. 
https://github.com/emmebrusa/TSDZ2-Smart-EBike-1/blob/9f22efbac8baab6c2644afe1cefad0974b1be647/src/controller/ebike_app.c#L2986-L2987

In practice, if the battery voltage drops a bit, then the current value limit based on the battery power will increase a bit. Which is the opposite of what you want to happen, if you want to protect the cells.
Additionally, you will be getting a bit more torque at lower battery voltage (but slower max cadence nevertheless due to bemf and physics), which is not desired either. 

In conclusion, just set it to a high value (800), don't rely on it, and protect the battery and the motor using Battery **Current** Max. 

Someone, please let me know if I missed something :)
